src/textchannellistener.cpp

```diff
@@ -1665,15 +1665,6 @@ void TextChannelListener::channelReady()
                        TP_QT_IFACE_CHANNEL_INTERFACE_GROUP)) {
             m_IsGroupChat = true;
             m_GroupHandleType = Tp::HandleTypeNone;
-            m_PersistentId = m_Channel->immutableProperties().value(
-                TELEPATHY_CHANNEL_INTERFACE_PERSISTENT_ID).toString();
-            qCDebug(lcCommhistoryd) << Q_FUNC_INFO << "group chat: HandleTypeNone, PersistentId ="
-                     << m_PersistentId;
-
-            if (m_PersistentId.isEmpty()) {
-                qCritical() << Q_FUNC_INFO << "No persistent id for Tp::HandleTypeNone groupchat";
-                return;
```

Comment:
This maybe slight behavioral change. If there hasn't been persistent ids available this should have always existed. Then again I'm not entirely following why the code would be interested in having the persistent id if it's not used for anything.